index.js

```diff
@@ -1,19 +1,30 @@
-var SpeechToTextV1 = require('watson-developer-cloud/speech-to-text/v1')
 var fs = require('fs')
 var express = require('express')
 var bodyParser = require('body-parser')
 var multer = require('multer')
+var convertToWav = require('./convert-to-wav')
+var deleteWav = require('./delete-wav')
+var callWatson = require('./call-watson')
 
 var app = express()
-
 app.use(express.static('public'))
 app.use(bodyParser.json())
 
-var upload = multer({ dest: 'upload/'})
+var upload = multer({ dest:'tmp/' })
 
-app.post('/upload', upload.single('file'), (req,res) => {
+app.post('/tmp', upload.single('file'), (req,res) => {
```

Comment:
this will ultimately be your `command` or `search` route for your API, so change it to something more appropriate